src/modules/ToDoList.js

```diff
@@ -0,0 +1,50 @@
+export default class ToDoList {
+  constructor(taskList = []) {
+    this.taskList = taskList;
+  }
+
+  get getTaskList() {
+    return this.taskList;
+  }
+
+  set setTaskList(newTaskList = []) {
+    if (!newTaskList) return;
+    this.taskList = newTaskList;
+  }
+
+  getLength() {
+    return this.taskList.length;
+  }
+
+  addTask(task) {
+    this.taskList.push(task);
+  }
+
+  removeTask(index) {
+    this.taskList.splice(index, 1);
+    this.updateTaskIndex();
+  }
+
+  updateTaskIndex() {
+    this.taskList.sort((a, b) => a.index - b.index);
+    for (let i = 0; i < this.taskList.length; i += 1) {
+      this.taskList[i].index = i;
+    }
+  }
```

Comment:
- [ ] **[OPTIONAL]** Good job implementing the changes by the previous code reviewer 👍🏽 When a new task is added to the list the index is the same in the array index, take in mind this project requirement and kindly fix it to complete all the project requirements 👇🏽 

> ...the property index set to the value of the new array length (i.e. **if you're adding a 5th task to the list, the index of that task should equal to 5**).

![image](https://user-images.githubusercontent.com/58642949/208218851-03f90528-9f1a-47c0-83db-f3f943ccf9b3.png)